### PR TITLE
3052 refactor/top right nav update

### PR DESCRIPTION
**Pull request by perryr16**
#### Any background context you want to provide?
See issue #3050

#### What's this PR do?
- This PR updates the function and appearance of the `_header` navigation buttons. 
- The `+` button will now direct users to the organization settings of the current organization based on their authorization role 
- The current organization has been reformatted to show the current (selected) organization
- When the user hovers over an organization in the drop-down the organization id will appear above the mouse

#### How should this be manually tested?
- From any page, use the `+` button to navigate to the organization's various settings. Ensure it is the correct organization (check URL for organization id) 
- with an Admin account (role_level = 20) see all child organizations
- with a nonauthorized user check the available settings and organizations. If they do not have permission (role_level = 0) the `+` button will disappear. Partially authorized users (role_level = 10) will be able to click the `+` but will only have access to 'members' and 'more...'
- Hovering over the organization dropdown will show the organization id for all users

#### What are the relevant tickets?
#3052 

#### Admin  (`role_level` = 20) hover over organization dropdown 
Screen Shot 2022-01-05 at 10 23 55 AM

Screen Shot 2022-01-05 at 10 23 45 AM

#### Admin (`role_level` = 20) `+` functionality
Screen Shot 2022-01-05 at 10 23 24 AM

#### User (`role_level` = 10) `+` functionality
Screen Shot 2022-01-05 at 10 24 57 AM

#### User (`role_level` = 10) hover over organization dropdown
Screen Shot 2022-01-05 at 10 47 45 AM

#### Non-authorized user (`role_level` = 0) hover over organization dropdown. 
Note: `+` hidden from non-authorized user
Screen Shot 2022-01-05 at 10 48 15 AM

### CODE REVIEW

File: seed/static/seed/scss/style.scss [hunk 1]
```diff
@@ -367,6 +367,22 @@ a:not([href]) {
...
+        z-index: 10000;
+        width: 100px;
+      }
+
+      .hover-org-id {
+        font-size: 10px;
+        background:rgba(255, 255, 255, 0.8);
+        border: 1px solid gray;
+        border-radius: 3px;
+        color: gray;
```
Comment (perryr16) [inline seed/static/seed/scss/style.scss:382]: is gray too hard to read?

File: seed/templates/seed/_header.html [hunk 2]
```diff
@@ -20,16 +31,22 @@
...
+                <li><a ui-sref="organization_sharing({organization_id: org.id})" ng-if="org.is_parent && auth.requires_owner" ui-sref-active="active" translate>Sharing</a></li>
+                <li><a ui-sref="organization_column_settings({organization_id: org.id, inventory_type: 'properties'})" ng-if="auth.requires_owner" ui-sref-active="active" translate>Column Settings</a></li>
+                <li><a ui-sref="organization_data_quality({organization_id: org.id, inventory_type: 'properties'})" ng-if="org.is_parent && auth.requires_owner" ui-sref-active="active" translate>Data Quality</a></li>
+                <li><a ui-sref="organization_cycles({organization_id: org.id})" ng-if="org.is_parent && auth.requires_owner" ui-sref-active="active" translate>Cycles</a></li>
+                <li><a ui-sref="organization_labels({organization_id: org.id})" ng-if="auth.requires_owner" ui-sref-active="active" translate>Labels</a></li>
+                <li><a ui-sref="organization_members({organization_id: org.id})" ui-sref-active="active" translate>Members</a></li>
+                <li><a ui-sref="organization_members({organization_id: org.id})" translate>More...</a></li>
```
Comment (perryr16) [inline seed/templates/seed/_header.html:44]: `More...` sends users to `.../members`. This was pulled from the organization link in `.../accounts`

**Review by macintoshpie — COMMENTED** [review]
Works like a charm! I did notice though that the tooltip hung around after I'd click on an org (which we should probably get rid of):
Screen Shot 2022-01-06 at 1 02 51 PM

The only other feedback I have is that we should probably get rid of the "More..." option since it's redundant (links to Members)

File: seed/static/seed/scss/style.scss (hunk 1 above)
Comment (macintoshpie) [reply seed/static/seed/scss/style.scss:382]: I think it's fine

**Review by macintoshpie — APPROVED** [review]
💯 ⛷️

**Pull request merged**